build/checkversion.py

```diff
@@ -37,7 +37,7 @@ def changelog_version():
   """Gets the version of the library from the CHANGELOG."""
   path = os.path.join(shakaBuildHelpers.get_source_base(), 'CHANGELOG.md')
   with shakaBuildHelpers.open_file(path, 'r') as f:
-    match = re.search(r'## (.*) \(', f.read())
+    match = re.search(r'### \[(.*?)\]\(', f.read())
```

Comment:
This can sometimes be just ## rather than ###.
For example: https://github.com/google/shaka-player/pull/3945/files

Reply:
Huh... I'm not sure why the release tool used two in one version and three in another.  I'll fix it to cover both.

Reply:
I assume it's because that's a minor version, rather than a bugfix version?

Reply:
Actually, if that's right, I wonder if the tool will use a single # if we release 4.0.0

Reply:
Yeah, I think you may be right.  I'll need to test the release workflow more thoroughly in my fork.